producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/MBeanConfig.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.moj.sdt.producers.comx.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jmx.export.MBeanExporter;
+import org.springframework.jmx.export.assembler.MethodNameBasedMBeanInfoAssembler;
+import org.springframework.jmx.support.RegistrationPolicy;
+import uk.gov.moj.sdt.utils.mbeans.SdtMetricsMBean;
+
+@Configuration
+@EnableAutoConfiguration
+public class MBeanConfig {
+
+    @Bean
+    @Qualifier("sdtProducersMetrics")
```

Comment:
Should the qualifier name be "sdtProducersComxMetrics" so it matches the value on line 24?

Reply:
changed to sdtProducersComxMetrics